generators/app/__snapshots__/generator.spec.mts.snap

```diff
@@ -1144,7 +1144,7 @@ exports[`generator - app with microservice should match snapshot 1`] = `
   "searchEngineCouchbase": false,
   "searchEngineElasticsearch": false,
   "searchEngineNo": true,
-  "serverPort": 8081,
+  "serverPort": 8080,
```

Comment:
IFAIK microservice port should be 8081 by default

Reply:
why this small change in the code can lead to that side effect? Looked 3 times at the logic, it shouldn't impact this snapshot. Also I'm not sure that the port should be 8081 as there's no gateway described in the test.

Reply:
Updated, thank you very much for the review!